pypeit/images/combineimage.py

```diff
@@ -64,8 +61,8 @@ def run(self, process_steps, bias,
 
         Args:
             process_steps (list):
-            bias (np.ndarray or None):
-                Bias image or instruction
+            bias (np.ndarray, optional):
+                Bias image
```

Comment:
please fix docs. this is now a single flatimages object. Explain what it is.

Reply:
updated